kano_avatar/character_components.py

```diff
@@ -177,6 +195,13 @@ def paste_over_image(self, img):
         transp_mask = Image.merge("L", (a,))
         img.paste(item, position, transp_mask)
 
+        # repeat for overworld image, but no offset
```

Comment:
I don't think we ended up using the x, y offset while pasting anyway. At some point it should be removed because adds a lot of junk to the configuration files :-)